app/views/cost_items/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="form-group <%= 'form-group--error' if f.object.errors[:name].any? %>">
```

Comment:
Tempting to extract a helper with a block for all the `errors[:field]` checking here...